namespaces/live.cloud-platform.service.justice.gov.uk/whereabouts-api-dev/resources/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
 }
 
 provider "aws" {
-  region = "eu-west-1"
+  region = "eu-west-2"
```

Comment:
Set default region to match other resources, as was not able to find SSM param in `eu-west-1`